lib/spack/spack/environment/environment.py

```diff
@@ -1399,7 +1399,9 @@ def _concretize_separately(self, tests=False):
 
         # Solve the environment in parallel on Linux
         start = time.time()
-        max_processes = min(len(arguments), 16)  # Number of specs  # Cap on 16 cores
+
+        # Number of processes, capped to config:build_jobs & # specs.
```

Comment:
```suggestion
        # Number of processes, capped to config:build_jobs or 16.
```

Reply:
It's actually `# specs` too (that is `len(arguments)`). I think the literal 16 doesn't really need a comment ;p

Reply:
Ah, got it, thanks.